sherlock/internal/pactbroker/record_deployment.go

```diff
@@ -0,0 +1,43 @@
+package pactbroker
+
+import (
+	"errors"
+	"fmt"
+	"github.com/broadinstitute/sherlock/sherlock/internal/config"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"net/http"
+)
+
+const pburl = `https://pact-broker.dsp-eng-tools.broadinstitute.org`
+
+// Record deployment to pact broker
+// https://docs.pact.io/pact_broker/recording_deployments_and_releases
+func RecordDeployment(chartName string, appVersion string, eID uuid.UUID) {
+	if config.Config.Bool("pactbroker.enabled") {
+
+		request, err := http.NewRequest(http.MethodPost, pburl+"/pacticipants/"+chartName+
+			"/versions/"+appVersion+"/deployed-versions/environment/"+eID.String(), nil)
+		if err != nil {
+			PactSwallowErrors(err)
```

Comment:
```suggestion
			return swallowErrors(err)
```

Reply:
I think when we swallow an error we should exit, instead of keep going with the rest of the function.